index.php

```diff
@@ -28,6 +28,18 @@
     }
 }
 
+//Checks if a POST is set for newFavourite and breedID.
+//Then updates row in Database setting the breed row to have that newFavourite ID.
+if (isset($_POST['newFavourite']) && isset($_POST['breedID'])) {
+    $favouriteSql = $dbConnection->prepare('UPDATE `breed` SET favourite_id = :newFavourite WHERE id= :breedID;');
```

Comment:
I think it's worth leaving a comment specifying what favourite_id is in the table (it's the index id)